pipeline/splicing/test_checkVarHGVS.py

```diff
@@ -0,0 +1,289 @@
+import unittest
+import checkVarHGVS
+
+class testCheckVarHGVS(unittest.TestCase):
+
+    def setUp(self):
+
+        self.varBRCA1 = {"Ref":"C",
```

Comment:
best to use lowercase var names and spaces after the colons

Reply:
I didn't use lowercase var names for "Ref" and "Alt" because they are capitalized in the built.tsv file and wanted to be consistent with that file